packages/hawtio/src/plugins/camel/contexts/context-actions.component.ts

```diff
@@ -0,0 +1,104 @@
+/// <reference path="contexts.service.ts"/>
```

Comment:
Maybe it's a leftover from the legacy code?

Reply:
Shouldn't be there at all. Have committed an old angular file. Also spotted a WIP commit. Will scrub and rebase.